backend/service/service_account.go

```diff
@@ -35,9 +38,20 @@ func (service *AccountService) Query(address string) (result vo.AccountVo) {
 			}
 			result.Amount = amount
 		}
-		result.Deposits = delegatorService.GetDeposits(address)
+		//result.Deposits = delegatorService.GetDeposits(address)
+		valaddress := utils.Convert(conf.Get().Hub.Prefix.ValAddr, address)
+		prefix, _, _ = utils.DecodeAndConvert(valaddress)
+		if prefix == conf.Get().Hub.Prefix.ValAddr {
```

Comment:
there is no need to get prefix of valaddress

Reply:
have done